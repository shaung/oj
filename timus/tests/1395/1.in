9
1 3 5 11 2 3 4 7 9
