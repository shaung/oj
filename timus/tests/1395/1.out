maxlength=6, starts with 1, 3
